scss/civicrm/administer/settings/_resource-urls.scss

```diff
@@ -9,13 +9,15 @@
 
 
     table.form-layout {
-      @include civicrm-panel;
-
       td:first-of-type {
         width: 30%;
       }
     }
 
+    input[type=radio] + label {
+      line-height: 135%;
```

Comment:
this looks extremely weird and specific

Reply:
Removed based on new changes of radio/checkboxes.